jiant/pytorch_transformers_interface/modules.py

```diff
@@ -0,0 +1,309 @@
+import copy
+import logging as log
+import os
+from typing import Dict
+
+import torch
+import torch.nn as nn
+from allennlp.modules import scalar_mix
+
+import pytorch_transformers
+
+from jiant.preprocess import parse_task_list_arg
+from jiant.utils import utils
+
+
+class PytorchTransformersEmbedderModule(nn.Module):
+    """ Shared code for pytorch_transformers wrappers.
+
+    Subclasses share a good deal of code, but have a number of subtle differences due to different
+    APIs from pytorch_transfromers.
+    """
+
+    def __init__(self, args):
```

Comment:
nit: this only seems to depend on `args.exp_dir` and `args.pytorch_transformers_embedding_mode` - for cleaner abstractions, can we make this explicit?

This should work:
```
def __init__(self, exp_dir, pytorch_transformers_embedding_mode, **unused_kw):
  # do stuff
```
and call as `module(**args.to_dict())` 

(similarly with `parameter_setup()` below)

Reply:
The inheriting classes add a few more args, so this would either expand the code a fair bit or else add an inconsistency. Weak/lazy preference to leave as is for now.